src/main/java/zmaster587/advancedRocketry/atmosphere/AtmosphereSuperHighPressure.java

```diff
@@ -0,0 +1,81 @@
+import net.minecraft.item.ItemStack;
+import net.minecraft.potion.Potion;
+import net.minecraft.potion.PotionEffect;
+import zmaster587.advancedRocketry.api.AdvancedRocketryItems;
+import zmaster587.advancedRocketry.api.EntityRocketBase;
+import zmaster587.advancedRocketry.api.armor.IFillableArmor;
+import zmaster587.advancedRocketry.api.armor.IProtectiveArmor;
+import zmaster587.advancedRocketry.integration.CompatibilityMgr;
+import zmaster587.advancedRocketry.network.PacketOxygenState;
+import zmaster587.libVulpes.LibVulpes;
+import zmaster587.libVulpes.network.PacketHandler;
+
+public class AtmosphereSuperHighPressure extends AtmosphereType {
+	
+	public AtmosphereSuperHighPressure(boolean canTick, boolean isBreathable,
+			String name) {
+		super(canTick, isBreathable, name);
+	}
+	
+
+	@Override
+	public String getDisplayMessage() {
+		return LibVulpes.proxy.getLocalizedString("msg.superHighPressure");
+	}
+	
+	@Override
+	public void onTick(EntityLivingBase player) {
+		if(player.worldObj.getTotalWorldTime() % 20  == 0 && !isImmune(player)) {
+			if(!isImmune(player)) {
+				player.addPotionEffect(new PotionEffect(Potion.digSlowdown.id, 40, 3));
+				player.addPotionEffect(new PotionEffect(Potion.moveSlowdown.id, 40, 3));
+				player.addPotionEffect(new PotionEffect(Potion.blindness.id, 40, 1));
+				player.attackEntityFrom(AtmosphereHandler.oxygenToxicityDamage, 1);
+				if(player instanceof EntityPlayer) {
+					PacketHandler.sendToPlayer(new PacketOxygenState(), (EntityPlayer)player);
+				}
+			}
+		}
+	}
+	
+	
+	@Override
+	public boolean isImmune(EntityLivingBase player) {
+
+		//Checks if player is wearing spacesuit or anything that extends ItemSpaceArmor
+		ItemStack feet = player.getEquipmentInSlot(1);
+		ItemStack leg = player.getEquipmentInSlot(2);
+		ItemStack chest = player.getEquipmentInSlot(3);
+		ItemStack helm = player.getEquipmentInSlot(4);
+
+		//TODO change over to use API #ISealedArmor
+		return (player instanceof EntityPlayer && ((EntityPlayer)player).capabilities.isCreativeMode) 
+				|| player.ridingEntity instanceof EntityRocketBase ||
+				helm != null && (helm.getItem() instanceof IProtectiveArmor && ((IProtectiveArmor)helm.getItem()).protectsFromSubstance(this, helm, true) || protectsFrom(helm, 1)) &&
+				chest != null && (chest.getItem() instanceof IProtectiveArmor && ((IProtectiveArmor)chest.getItem()).protectsFromSubstance(this, chest, true) || protectsFrom(chest, 2)) &&
+				leg != null && (leg.getItem() instanceof IProtectiveArmor && ((IProtectiveArmor)leg.getItem()).protectsFromSubstance(this, leg, true) || protectsFrom(leg, 3)) &&
+				feet != null && (feet.getItem() instanceof IProtectiveArmor && ((IProtectiveArmor)feet.getItem()).protectsFromSubstance(this, feet, true) || protectsFrom(feet, 4)) &&
+				(protectsFrom(chest, 2) || ((chest.getItem() instanceof IFillableArmor) && ((IFillableArmor)AdvancedRocketryItems.itemSpaceSuit_Chest).decrementAir(chest, 1) > 0));
```

Comment:
Do we really want to decrement O2 on a high pressure world?

Reply:
Hmm.... Does the space suit have an inbuilt outside O2 -> internal O2 filter? Super high pressure simulates O2 toxicity, so if it can pull O2 from outside and separate it then it would not be needed.

Reply:
I think until there's a suit module for it made, I think for all intents and purposes it'd be best to kinda assume there is a built in filter or some kind of pressure control system.  I think unless there's something fairly obvious, most players will be like "why is my O2 draining?  There's an atmosphere here"

Reply:
Great! Can do.

Reply:
Although it looks like standard high pressure atmospheres already decrement as well, do you want to keep it consistent between them, or change both?

Reply:
Done! VeryHot was added as well.